charts/jaeger-operator/COMPATIBILITY.md

```diff
@@ -0,0 +1,20 @@
+The following table shows the compatibility of jaeger operator chart with different components, in this particular case we shows jaeger-operator, Kubernetes and Strimzi operator compatibility.
+
+| Chart version          | Jaeger Operator | Kubernetes      | Strimzi Operator   | Cert-Manager |
+|------------------------|-----------------|-----------------|--------------------|--------------|
+| 2.32.*                 | v1.34.x         | v1.19 to v1.24  | v0.23              | v1.6.1       |
+| (X)                    | v1.33.x         | v1.19 to v1.23  | v0.23              | v1.6.1       |
+| 2.30.0(C), 2.31.0      | v1.32.x         | v1.19 to v1.21  | v0.23              | v1.6.1       |
+| 2.29.0(C)              | v1.31.x         | v1.19 to v1.21  | v0.23              | v1.6.1       |
+| 2.28.0                 | v1.30.x         | v1.19 to v1.21  | v0.23              |              |
+| 2.27.1                 | v1.29.x         | v1.19 to v1.21  | v0.23              |              |
+| 2.27.0                 | v1.28.x         | v1.19 to v1.21  | v0.23              |              |
+| 2.26.0                 | v1.27.x         | v1.19 to v1.21  | v0.23              |              |
+| (X)                    | v1.26.x         | v1.19 to v1.21  | v0.23              |              |
+| (X)                    | v1.25.x         | v1.19 to v1.21  | v0.23              |              |
+| 2.23.0, 2.24.0, 2.25.0 | v1.24.x         | v1.19 to v1.21  | v0.23              |              |
+| (X)                    | v1.23.x         | v1.19 to v1.21  | v0.19, v0.20       |              |
+| 2.21.\*                | v1.22.x         | v1.18 to v1.20  | v0.19              |              |
+Legend:
+- `(C)` Chart is corrupted. Please do not use it, see [link](https://github.com/jaegertracing/helm-charts/issues/351)
+- `(X)` Missing chart version
```

Comment:
Maybe could make this a little more clear, along the lines of "Missing chart version for specified Jaeger Operator version"

On first glance I was confused by what determined a missing version.

Reply:
Yeah, there is plenty things going on here, that's why I wanted add a little explanation.  I've updated and squashed matrix. Is there anything we can add/remove? I am open to any suggestions.

Reply:
I think other than that commend on the cert manager it looks good to me.